shared/src/main/scala/mlscript/ucs/Desugarer.scala

```diff
@@ -375,8 +375,7 @@ class Desugarer extends TypeDefs { self: Typer =>
         case L(IfBlock(lines)) =>
           lines.foreach { desugarMatchBranch(scrutinee, _, partialPattern, collectedConditions) }
         // This case is rare. Let's put it aside.
-        case L(IfLet(_, _, _, _)) =>
-          lastWords("please add this rare case to test files")
+        case L(IfLet(_, _, _, _)) => ???
```

Comment:
You know it's a pity that `???` does not have more info attached to it. You could add a method `def TODO(msg: Str) = throw new NotImplementedError(msg)` in `utils/package.scala`, and make this:
```diff
- case L(IfLet(_, _, _, _)) => ???
+ case L(IfLet(_, _, _, _)) => TODO("please add this rare case to test files")
```

Reply:
This is really cool. I added them.